### PR TITLE
Added Ability for simplepath to Deal with non-Iterable Object Data

### PR DESCRIPTION
**Pull request by NdubisiOnuora**
@miki725, can you review and merge this PR?
This replaces "https://github.com/dealertrack/simplepath/pull/3" and matches the guidelines to have branches contain changes in their names.

### CODE REVIEW

**Comment by NdubisiOnuora** [comment]
@miki725, build has passed.

File: tests/test_z_integration.py [hunk 1]
```diff
@@ -61,3 +61,54 @@ class Config(Mapper):
...
+        data = Example("Bonjour", planets)
+
+        expected = {
+            'greetings': 'Bonjour',
+            'from': 'friends',
+            'to': 'people',
+            'neighbors': [
+                {
+                    'from': 'Mars',
+                    'neighbors': 'marsians',
```
Comment (gregarmer) [inline tests/test_z_integration.py]: That should be `martians` :wink:

Comment (NdubisiOnuora) [inline tests/test_z_integration.py]: @gregarmer, respected the Martians.
Respect

File: tests/test_lookups.py [hunk 2]
```diff
@@ -87,6 +87,17 @@ def test_call_dict(self):
 
         self.assertEqual(actual, 'bar')
 
+    def test_call_object(self):
+        """Ensure that we can lookup values in non iterable objects.
+        """
```
Comment (miki725) [inline tests/test_lookups.py:92]: thanks for the docstring!

can you please move this ending triple quote at the end of last line as per pep8:

> For one liner docstrings, please keep the closing """ on the same line.

[source](https://www.python.org/dev/peps/pep-0008/)

**Comment by miki725** [comment]
@NdubisiOnuora just a small pep8 comment. rest LGTM.

also please add your name to contributors list

**Comment by NdubisiOnuora** [comment]
Closed and replaced with "https://github.com/dealertrack/simplepath/pull/5"

**Pull request closed without merging**